x-pack/plugins/translations/translations/fr-FR.json

```diff
@@ -15664,6 +15664,15 @@
     "xpack.infra.homePage.noMetricsIndicesInstructionsActionLabel": "Voir les instructions de configuration",
     "xpack.infra.homePage.settingsTabTitle": "Paramètres",
     "xpack.infra.homePage.toolbar.kqlSearchFieldPlaceholder": "Rechercher des données d'infrastructure… (par exemple host.name:host-1)",
+    "xpack.infra.hostsTable.nameColumnHeader": "Nom",
+    "xpack.infra.hostsTable.operatingSystemColumnHeader": "Système d'exploitation",
+    "xpack.infra.hostsTable.numberOfCpusColumnHeader": "Nombre de processeurs",
+    "xpack.infra.hostsTable.diskLatencyColumnHeader": "",
```

Comment:
Note: Those translations are missing (couldn't find them in the project so I guess they should be added)